CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+## Forward
+
+So, you're looking to contribute to Fabric API. Well, you've come to the right place. This guide is dedicated to teaching you everything you need to know about contributing to Fabric API. This guide might be kind of lengthy, but there's a lot of useful information in it, so make sure to read until the end.
+
+Before we begin, some friendly advice:
+1. We don't tend to ban people just for having ideas, no matter how outlandish. However, “no” can mean “no”, and un-constructive responses to constructive arguments can get on our nerves after some time. We are generally unpaid volunteers who dedicate a lot of free time to the tiring job of social interaction - please don't make our work harder than it already is.
+2. We're here to help each other learn, after all. Don't hesitate to talk to us at any point! [The Fabric Discord](https://discord.gg/v6v4pMv) is a great place to ask questions and get input on your contributions.
+3. Fabric isn't easy to get your content into, as we uphold standards of quality and are very paranoid about making a bad precedent. You will be scrutinized again, and again, and again; you will be asked questions about more-or-less every line in your pull request (even the imports!).
```

Comment:
Second sentence here probably isn't necessary. Paranoia probably isn't the intent either.

I would probably word this along the line of, "Contributions to Fabric API are expected to hold a certain standard of quality. These standards are justified in order to provide a reasonably stable API for mods to build off of. You should be prepared to answer questions related to features you propose and the implementation detail of said features."